scripts/statesync.bash

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# microtick and bitcanna contributed significantly here.
+# Pebbledb state sync script for evmos
+set -uxe
+
+# Set Golang environment variables.
+export GOPATH=~/go
+export PATH=$PATH:~/go/bin
+
+# Install Juno with pebbledb 
```

Comment:
```suggestion
# Install fetchd with pebbledb 
```

Reply:
What is pebbledb? I don't think it's required to have it but curious of its advantages over default db